pkg/lib/interaction/nodes/create_authenticator_begin.go

```diff
@@ -58,9 +58,7 @@ func (n *NodeCreateAuthenticatorBegin) Prepare(ctx *interaction.Context, graph *
 		return err
 	}
 
-	if n.Stage == interaction.AuthenticationStagePrimary {
```

Comment:
Would like to know why don't we need this checking any more?
Would creating mfa in setting page go through this node, in that case we may not have identity? Or setting is different?

Reply:
You're correct! Identity may be nil if authenticator is being added in settings page. However, it means that we have to hide MFA section in the settings page if the user has only one OAuth identity. This is related to the REDACTED project's requirement though.